### DISCUSSION TITLE
Spacy Dependency Versions: Why Are Older Versions of Typer and Weasel Being Installed?

### DISCUSSION THREAD

**Discussion by avivtp** [explosion/spaCy#13507] [Help: Installation]
Hi,

I'm trying to install the latest version of `spacy` via `pip`, but for some reason, it installs `typer v0.9` and `weasel v0.3`. 
According to the requirements file, newer versions of these dependencies should also be compatible.

Did the changes made in the `requirements.txt` and `setup.cfg` files not make it into the latest release?
If not, when should we expect to get version 3.7.5?

Thank you!

**Comment by BLKSerene** [comment]
The master branch contains commits made after the release of `spaCy` 3.7.4, including the extending of `typer` and `weasel` to latest versions. It seems that development team has already bumped version to 3.7.5 but hasn't release it to PyPI.

**Reply by avivtp** [reply]
Thank you for your response. Do you know if the team plans to release version 3.7.5 to PyPI soon?

**Reply by BLKSerene** [reply]
I have no idea. Maybe you could directly ask `spaCy` team member.
